tests/test_sanity.py

```diff
@@ -584,13 +576,12 @@ def test_emconfig(self):
 
     temp_dir = tempfile.mkdtemp(prefix='emscripten_temp_')
 
-    os.chdir(temp_dir)
-    self.do([PYTHON, EMCC, '--em-config', custom_config_filename] + MINIMAL_HELLO_WORLD + ['-O2'])
-    result = run_js('a.out.js')
-    
+    with chdir(temp_dir):
+      self.do([PYTHON, EMCC, '--em-config', custom_config_filename] + MINIMAL_HELLO_WORLD + ['-O2'])
+      result = run_js('a.out.js')
+
     # Clean up created temp files.
     os.remove(custom_config_filename)
```

Comment:
Should this be scoped to the chdir(temp_dir), or is the custom_config_filename an absolute path?

Reply:
No its absolute.